<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chinese chess</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/Resource.qrc">
    <normaloff>:/image/Chess</normaloff>:/image/Chess</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="actionPrevious_P"/>
    <addaction name="actionNext_N"/>
    <addaction name="actionPrompt_message_M"/>
    <addaction name="actionPrompt_sound_S"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionUpdate_U"/>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_O"/>
   <addaction name="actionSave_S"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious_P"/>
   <addaction name="actionNext_N"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionPrevious_P">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Previous(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>Previous(P)</string>
   </property>
   <property name="statusTip">
    <string>Previous(P)</string>
   </property>
   <property name="whatsThis">
    <string>Previous(P)</string>
   </property>
  </action>
  <action name="actionNext_N">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next(&amp;N)</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Chess</normaloff>:/image/Chess</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
  <action name="actionPrompt_message_M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prompt message(&amp;M)</string>
   </property>
  </action>
  <action name="actionPrompt_sound_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prompt sound(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
